### PR TITLE
CHORE: DTO, Repository and Profile added

### PR DESCRIPTION
**Pull request by VictorRosaValdez**
DTO, Repository and Profile added.

### CODE REVIEW

**Review by Leroy96 — APPROVED** [review]
Ziet er goed uit! Alleen veel onnodige witregels, maar dat kunnen we er later uithalen.

File: DOTNET_Final_Case_BackEnd/Program.cs [hunk 1]
```diff
@@ -38,9 +39,17 @@
...
+// Injection of the DbContext.
 builder.Services.AddDbContext<ProjectsDbContext>(x => x.UseSqlServer(connectionString));
+
+// Injection of AutoMapper.
 builder.Services.AddAutoMapper(typeof(Program));
 
+
+
+
+
```
Comment (Pimbosly) [inline DOTNET_Final_Case_BackEnd/Program.cs]: feels a little much whitelines

**Review by Pimbosly — CHANGES_REQUESTED** [review]
redelijk wat whitelines bij de interfaces en profiles.
Code ziet er voor de rest goed uit.

**Review by Dorine-forritari — COMMENTED** [review]
Great work! I think in the ProjectCreateDTO and ProjectReadDTO we should make Industry a compulsory field instead of nullable (so we always know which Industry a project belong to and the design will also depend on that with the icons). Maybe for the Description as well, what do you guys think? And in the repositories in the comments 'Abstract method for ...' -> ' Abstract method to....' (English grammar ;))

File: DOTNET_Final_Case_BackEnd/Program.cs (hunk 1 above)
Comment (VictorRosaValdez) [reply DOTNET_Final_Case_BackEnd/Program.cs]: White lines were deleted.

**Comment by VictorRosaValdez** [comment]
Feedback processed.

**Pull request merged**